docs/creatingdatabases.md

```diff
@@ -41,6 +44,21 @@ spec:
   backup:
     enable: false # turn it to true when you want to use back up feature. currently only support postgres
     cron: "0 0 * * *"
+  connectionStringTemplate: |
+    "jdbc:{{ .Protocol }}://{{ .UserName }}:{{ .Password }}@{{ .DatabaseHost }}:{{ .DatabasePort }}/{{ .DatabaseName }}" # provide a custom template to generate a database connection string
+```
+If your application needs connections string in another format, you can provide your own template via the connectionStringTemplate field. When the `connectionStringTemplate` the default template is being used:
```

Comment:
`When the `connectionStringTemplate` the default template is being used` this sentence doesn't seem complete. did you want to say `When the `connectionStringTemplate` is empty, the default template is being used` ?

Reply:
Yeah, I'm sorry. That's exactly what I wanted to say. Fixed now